### PR TITLE
fix(l10n): fix translations for the password warning

### PR DESCRIPTION
**Pull request by vladikoff**

### CODE REVIEW

**Comment by vladikoff** [comment]
cc @vbudhram

**Comment by vladikoff** [comment]
Fixes the problem described in https://github.com/mozilla/fxa-content-server/pull/4065

**Comment by vbudhram** [comment]
@vladikoff LGTM 👍 , r+!

screen shot 2016-08-22 at 5 00 20 pm

**Comment by vladikoff** [comment]
thanks @vbudhram !

File: app/scripts/views/mixins/password-prompt-mixin.js [hunk 1]
```diff
@@ -56,6 +56,8 @@ define(function (require, exports, module) {
         promptContent = TOOLTIP_MESSAGES.WARNING_PROMPT_MESSAGE_WITH_LINK;
       }
 
+      promptContent = this.translate(promptContent);
```
Comment (l-hedgehog) [inline app/scripts/views/mixins/password-prompt-mixin.js]: This one is not really necessary, `Tooltip` will translate the string.

Comment (vladikoff) [inline app/scripts/views/mixins/password-prompt-mixin.js]: thanks!

**Pull request merged**